Miscellaneous/Optimization Techniques/readme.md

```diff
@@ -0,0 +1,6 @@
+## Optimization Techniques
```

Comment:
Currently, there is no need to create a new section "Optimization Techniques". But, will sleep over it on ml-papers21.